src/components/container/MicroTrend/MicroTrend.tsx

```
@@ -0,0 +1,81 @@
+    const hasRecentData = Object.values(results).some(r => r.value > 0);
+    if (props.hideIfNoRecentData && !hasRecentData) {
+        return null;
+    }
+
+    const definition = getDailyDataTypeDefinition(props.dataType.dailyDataType);
+    const formatter = props.dataType.formatter ?? definition.formatter;
+    let formattedValue = "--";
+    let noData = true;
+     const todayKey = getDayKey(date);
+     const todayValue = results[todayKey]?.value ?? 0;
+     const noData = todayValue === 0;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'noData'. Consider to delete it or rename it.

'noData' is defined here:


(lint/suspicious/noRedeclare)